auth-web/src/App.vue

```diff
@@ -169,6 +169,11 @@ export default class App extends Mixins(NextPageMixin) {
     // Listen for event from signin component so it can initiate setup
     this.$root.$on('signin-complete', async (callback) => {
       await this.setup()
+      // set logout url ONLY for bcros user
+      if (sessionStorage.getItem(SessionStorageKeys.UserAccountType) === LoginSource.BCROS) {
+        const logoutUrl = `${window.origin}/${process.env.VUE_APP_PATH}signin/bcros`
```

Comment:
We can use the /signout/:redirectUrl route for this, since the Signout component in the common components manages the redirection there.